### PR TITLE
MTA-46: Maven configuration changes

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-46
MTA 6.0.1
Preview: https://deploy-preview-655--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#configuring-maven-repositories

### CODE REVIEW

File: docs/topics/mta-web-config-maven-repo-size.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 You can reduce the size of a Maven repository in the *Repositories* view of the {ProductName} ({ProductShortName}) {WebName}.
 
+[NOTE]
+====
+This procedure is possible only if the `rwx_supported` configuration option of the Tackle CR is set to `true`. Otherwise, both the *Local artifact repository* field and the *Clear repository* button are disabled.
+====
```
Comment (ibragins) [inline docs/topics/mta-web-config-maven-repo-size.adoc:14]: This is NOT correct. `rwx_supported` parameter can be missed and absent in the CR and still field and button will be ENABLED. 
To disable them - `false` value should be set.

File: docs/topics/mta-web-config-maven-repo.adoc [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 You can configure a Maven repository in the *Repositories* view of the {ProductName} ({ProductShortName}) {WebName}.
 
+[NOTE]
+====
+This procedure is possible only if the `rwx_supported` configuration option of the Tackle CR is set to `true`. Otherwise, the *Consume insecure artifact repositories* switch is disabled.
```
Comment (ibragins) [inline docs/topics/mta-web-config-maven-repo.adoc:13]: Need to double check this, if I remember correctly - this is not affected by `rwx_supported` parameter value

Comment (sbeskin-redhat) [reply docs/topics/mta-web-config-maven-repo.adoc:13]: @ibragins 
The screenshot that you sent me shows that the switch is disabled.

File: docs/topics/mta-web-config-maven-repo-size.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-config-maven-repo-size.adoc]: @ibragins 
Corrected, please take a look.

**Review by ibragins — APPROVED** [review]
LGTM

**Comment by RichardHoch** [comment]
LGTM!

**Pull request merged**